File: Jongryul/04/1269.py
import sys
input = sys.stdin.readline

n, m = map(int, input().split())
a = set(map(int, input().split()))
b = set(map(int, input().split()))

cnt = n + m

for i in a:
    if i in b:
        cnt -= 2
print(cnt)

File: 11th_Competition/Jongryul/04/1269.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "input.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

const [n, m] = input[0].split(" ").map(Number);
const a = new Set(input[1].split(" ").map(Number));
const b = new Set(input[2].split(" ").map(Number));

let cnt = n + m;

for (i of a) {
  if (b.has(i)) {
    cnt -= 2;
  }
}
console.log(cnt);
